content/swiftui/concepts/viewmodifier/terms/foregroundColor/foregroundColor.md.md

```diff
@@ -0,0 +1,48 @@
+﻿---
+Title: .foregroundColor()
+Description: The .foregroundColor() modifier sets the foreground color displayed in a view.
+Subjects:
+ - 'SwiftUI'
+Tags: 
+ - 'SwiftUI'
+ - 'Modifier'
+ - 'View'
+ CatalogContent:
+ - 'learn-swiftUI'
+ - docs/content/swiftui/concepts/viewmodifier/terms/foregroundColor
+---
+# .foregroundColor ()
+The `.foregroundColor()` modifier sets the foreground color displayed in a [View](https://www.codecademy.com/resources/docs/swiftui/views).
+
```

Comment:
Remove the extra line breaks. There should be only one line break before and after the header.